content/en/docs/demo/services/recommendation.md

```diff
@@ -93,4 +93,47 @@ of `opentelemetry-bootstrap` on building the recommendationservice Docker image:
 
 ## Logs
 
-TBD
+### Initializing Logs
+
+The OpenTelemetry SDK is initialized in the `__main__` code block. This code
+will create a logger provider with a batch processor, an OTLP log exporter, and
+a logging handler. Finally, a logger is created to be used for creating logs
+throughout the application.
+
+```python
+    logger_provider = LoggerProvider(
+        resource=Resource.create(
+            {
+                'service.name': service_name,
+            }
+        ),
+    )
+    set_logger_provider(logger_provider)
+    log_exporter = OTLPLogExporter(insecure=True)
+    logger_provider.add_log_record_processor(BatchLogRecordProcessor(log_exporter))
+    handler = LoggingHandler(level=logging.NOTSET, logger_provider=logger_provider)
+
+    logger = logging.getLogger('main')
+    logger.addHandler(handler)
+```
+
+### Create log records
+
+Log records are created by using a logger, which was previously attached to
+the logging handler. Examples can be found in ListRecommendation and 
+get_product_list functions.
```

Comment:
Earlier in the page `ListRecommendations` (note the "s") is used. Should is be plural here too?

I don't think it is useful to mention the handler here.

Here's a suggested edit:

```suggestion
Use the logger to create log records. Examples can be found in the `ListRecommendation` and 
`get_product_list` functions.
```

Reply:
> Earlier in the page `ListRecommendations` (note the "s") is used. Should is be plural here too?

Yes, it should be with "s".